packages/runtime-core/src/component.ts

```diff
@@ -475,6 +475,11 @@ export interface ComponentInternalInstance {
    * @internal
    */
   ut?: (vars?: Record<string, string>) => void
+  /**
+   * cleanup el of hoisted vnode
+   * @internal
+   */
+  hoistedCleanup?: (() => void)[] | null
```

Comment:
I'm curious what the reasoning is for storing functions here, rather than just storing the VNodes directly?

Reply:
My initial idea was to add a cleanup method to `instance` to clean up the data on the instance, but this was probably over-engineered and later changed to `hoistedCleanup`. Storing VNodes might be a better solution, I'll update it later.